app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -705,9 +712,34 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   }
 
   override fun setAlternateId(alternateId: Identifier) {
+    when (alternateId.type) {
+      Identifier.IdentifierType.BangladeshNationalId -> setAlternateIdTextField(alternateId)
+      Identifier.IdentifierType.EthiopiaMedicalRecordNumber -> setAlternateIdTextField(alternateId)
+      Identifier.IdentifierType.IndiaNationalHealthId -> setAlternateIdContainer(alternateId)
+      else -> throw IllegalArgumentException("Unknown alternate id: $alternateId")
+    }
+  }
+
+  private fun setAlternateIdTextField(alternateId: Identifier) {
```

Comment:
We're not hiding the visibility of `alternateIdLabel` and it's container. Can we hide them in `setAlternateIdTextField` function?

Reply:
Ah I see. This makes sense